policy/release/rhtap_jenkins_checks.rego

```diff
@@ -0,0 +1,66 @@
+#
```

Comment:
Consider `rhtap_jenkins.rego` for the filename and `rhtap_jenkins` for the package name. `checks` is somewhat redundant since every rego file is performing checks.